src/Components/DetailsPage/DetailsPage.js

```diff
@@ -30,24 +34,24 @@ const DetailsPage = ( ) => {
             </div>
             <div className="taxonomy">
               <h3>Characterisics:</h3>
-              <p> Hymenium Type: {characteristic.hymeniumType},</p>
-              <p> Stipe Character: {characteristic.stipeCharacter},</p>
-              <p> Spore Print Color {characteristic.sporePrintColor},</p>
-              <p> Mushroom Cap Shape: {characteristic.mushroomCapShape},</p>
-              <p> Hymenium Attachment: {characteristic.hymeniumAttachment}</p>
-              <p> Mushroom Ecological Type: {characteristic.mushroomEcologicalType}</p>
+              <p> Hymenium Type: {characteristics.hymeniumType},</p>
```

Comment:
reflects the query database now! awesome

Reply:
So cool!!!